trpc/queries/goals.ts

```diff
@@ -302,6 +310,7 @@ export const addCalclulatedGoalsFields = (goal: any, activityId: string) => {
         _isEditable,
         _lastEstimate,
         _shortId,
+        history,
```

Comment:
Add underscore to mark field as calculated

Reply:
this is unused field, will drop it